src/pages/home/report/EmojiPickerMenuItem.js

```diff
@@ -3,28 +3,46 @@ import PropTypes from 'prop-types';
 import {Pressable, Text} from 'react-native';
 import styles, {getButtonBackgroundColorStyle} from '../../../styles/styles';
 import getButtonState from '../../../libs/getButtonState';
+import Hoverable from '../../../components/Hoverable';
 
 const propTypes = {
     // The unicode that is used to display the emoji
     emoji: PropTypes.string.isRequired,
 
     // The function to call when an emoji is selected
     onPress: PropTypes.func.isRequired,
+
+    // Toggles whether this emoji is highlighted
+    onHover: PropTypes.func.isRequired,
+
+    // Whether this menu item is currently highlighted or not
+    isHighlighted: PropTypes.bool.isRequired,
```

Comment:
Just for future reference, what are the different scenarios where I should be using `isHighlighted` vs. `shouldHighlight` for this and other similar boolean props? 

@pecanoro @stitesExpensify

Reply:
IMO I would use `isHighlighted` which it's a variable that represents the current state of an element and I would use `shouldHighlight` when it's something that we want to apply to a future element/variable, for example, passing it as a parameter to a function.

Reply:
Thanks!